### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by yanisalbert** [0xCUB3/About-This-Hack#13]
Hi
On Ventura 13.2 not working.
![Capture d’écran 2023-02-07 à 12 45 58](https://user-images.githubusercontent.com/87071207/217236993-d80afe8c-156d-4f3b-933c-85f40bc0c913.png)
![Capture d’écran 2023-02-07 à 12 46 06](https://user-images.githubusercontent.com/87071207/217237013-b6a058df-ad4c-40bf-a9c9-9b80db7e4588.png)
![Capture d’écran 2023-02-07 à 12 46 10](https://user-images.githubusercontent.com/87071207/217237026-a5347d8b-4010-438a-a5fd-789e11e6e9b7.png)

**Comment by 0xCUB3** [comment]
Yep apple revoked the app for some people again. Nothing I can do at this point, sorry

**Comment by yanisalbert** [comment]
Ok. Thanks
